cernopendata/jsonschemas/records/record-v1.0.0.json

```diff
@@ -61,57 +66,71 @@
           "type": "array"
         }
       },
+      "required": ["description"],
       "type": "object"
     },
     "accelerator": {
+      "description": "The accelerator involved in the production of the data",
       "type": "string"
     },
     "authors": {
       "items": {
         "properties": {
           "affiliation": {
+            "description": "The affiliation of the author",
             "type": "string"
           },
           "ccid": {
+            "description": "The author's ccid, if applicable",
```

Comment:
The "if applicable" bit could apply to virtually all the fields... I think it is better to leave these out everywhere.